examples/coprocessor_java/robot/build.gradle

```diff
@@ -16,9 +16,20 @@ deploy {
     }
 }
 
+// Note - the robot code doesn't need these repositories.
+// However, the 'common' project does, and for some reason it won't pull
+// its dependencies unless the repository is listed here.  Keep that in mind.
+repositories {
+  mavenCentral() // Used for the GSON library (Google's JSON converter)
```

Comment:
This shouldn't be required. The common project has its own transitive dependencies that are included from it's own repositories {} block.

Reply:
You're probably right. this is the first solution I stumbled upon when trying to resolve this error (just remove the mavenCentral() line).  I know this occurs if a dependency doesn't already exist in gradle cache, but I do not know if it occurs otherwise.
https://www.screencast.com/t/BkQGeqAwbF3i

Reply:
You seem to be correct. This appears to be something to do with Gradle internally. I'm going to close this review and open a new one